src/test/suite/connectionController.test.ts

```diff
@@ -843,43 +905,26 @@ suite('Connection Controller Test Suite', function () {
       StorageVariables.GLOBAL_SAVED_CONNECTIONS
     );
 
-    assert.strictEqual(
-      !!workspaceStoreConnections,
-      true,
-      `Expected workspace store to have connections, found ${workspaceStoreConnections}`
-    );
-
+    expect(workspaceStoreConnections).to.not.be.empty;
```

Comment:
How do you feel about keeping assert for consistency sake (as long as we don't move to chai+mocha)? No strong feelings about it but thought I should call it out.

Reply:
Or maybe we can leave it as is - I just saw you moved the align workflows ticket into "Ready for work" so that might cover it?

Reply:
I'd like to update all of the assert of this file over to expect in the soon, so I'm thinking the lack of consistency is alright for now.